spark-atlas-connector/src/test/scala/com/hortonworks/spark/atlas/sql/InsertIntoHarvesterSuite.scala

```diff
@@ -57,6 +67,22 @@ class InsertIntoHarvesterSuite extends FunSuite with Matchers with BeforeAndAfte
 
     sparkSession.sql(s"CREATE TABLE $destinationHiveTblName (name string)")
     sparkSession.sql(s"CREATE TABLE $destinationSparkTblName (name string) USING ORC")
+
+    // multiple source tables
+    sparkSession.sql(s"CREATE TABLE $inputTable1 (a int, b string)")
+    sparkSession.sql(s"INSERT INTO $inputTable1 VALUES(1, 'str1')")
+    sparkSession.sql(s"CREATE TABLE $inputTable2 (c int, d string)")
+    sparkSession.sql(s"INSERT INTO $inputTable2 VALUES(1, 'str2')")
+    sparkSession.sql(s"CREATE TABLE $inputTable3 (e int, f string)")
+    sparkSession.sql(s"INSERT INTO $inputTable3 VALUES(1, 'str3')")
+    sparkSession.sql(s"CREATE TABLE $outputTable1 (a int, b string, c int, d string, e int, f string)")
+
+    // multiple destination tables
+    sparkSession.sql(s"create table $inputTable4 (id int, code string, salary int)")
```

Comment:
Could you use an upper case for all SQL keyword?

Reply:
I have created a new issue to track this: https://github.com/hortonworks-spark/spark-atlas-connector/issues/17

Reply:
I've updated this. Also, I have created a new issue to track this: #17.